include/wil/resource.h

```diff
@@ -1298,7 +1298,10 @@ namespace wil
     {
         template<typename T> void operator()(_Frees_ptr_opt_ T* toFree) const
         {
-            TDeleter(toFree);
+            if (toFree)

```

Comment:
many deleter functions, like `LocalFree` accept `nullptr`, so this adds an unneeded test, growing the amount of code generated. should we differentiate deleters that do or don't accept null so we can optimize the `LocalFree` case?

Reply:
I think you're right. Everybody using the standard deleters must have this problem fixed already (in that the various `function_deleter` specializations use methods that are OK with null-ness.  The `mapview_deleter` annotation was just wrong and has been fixed while this method is left alone.